images/win/scripts/Installers/Install-DotnetSDK.ps1

```diff
@@ -107,6 +107,9 @@ function RunPostInstallationSteps()
 
     # Update Run key to run a script at logon
     Set-ItemProperty -Path "HKLM:\Software\Microsoft\Windows\CurrentVersion\Run" -Name "DOTNETUSERPATH" -Value $cmdDotNet
+
+    # Clear list of local NuGet resources in temporary cache
+    Invoke-Expression "dotnet nuget locals all --clear"
```

Comment:
I think these changes won't help.

If you run `dotnet nuget locals all --list`, it will print
```
http-cache: C:\Users\VssAdministrator\AppData\Local\NuGet\v3-cache
global-packages: C:\Users\VssAdministrator\.nuget\packages\
temp: C:\Users\VssAdministrator\AppData\Local\Temp\NuGetScratch
plugins-cache: C:\Users\VssAdministrator\AppData\Local\NuGet\plugins-cache
```
All these directories are located under current user folder. We always re-create user profile during deployment.
Also no one of these folders exist on clean Hosted VM